examples/image_classifier/mnist/README.md

```diff
@@ -34,6 +34,42 @@ Run the commands given in following steps from the parent directory of the root
     curl http://127.0.0.1:8080/predictions/mnist -T examples/image_classifier/mnist/test_data/0.png
     ```
 
+# Profiling inference output
+
+To enable pytorch profiler, set the following environment variable.
+
+```
+export ENABLE_TORCH_PROFILER=true
+```
+
+Note: Ensure`enable_envvars_config` is set to `true` in torchserve config
```

Comment:
@shrinath-suresh is this gonna be added to the config.properties?

Reply:
I added a note here, since the env based setup doesnt work unless user sets `enable_envvars_config` to `true` in config.properties. 

I will add `config.properties` file in this example and update the instructions accordingly.

Reply:
@shrinath-suresh as mentioned in previous comment, you can add a setting to the config following [this example](https://github.com/pytorch/serve/pull/1319/files). This basically would not work for now.